ksql-cli/src/test/java/io/confluent/ksql/TestRunner.java

```diff
@@ -66,12 +66,13 @@ protected static void test(String command, TestResult expectedResult, boolean re
         finalResults.clear();
         finalResults.addAll(actualResult.data);
         return actualResult.data.containsAll(expectedResult.data);
-      }, 10000, "Did not get the expected result '" + expectedResult + ", in a timely fashion. Received " +
-              "following results instead: " + finalResults);
+      }, 10000, "Did not get the expected result '" + expectedResult + ", in a timely fashion.");
+    } catch (AssertionError e) {
+      System.err.println("CLI test runner command result mismatch expected: " + expectedResult + ", actual: " + finalResults);
```

Comment:
log it? or remove the syserr

Reply:
I can log it. Why remove syserr? The assertion failure prints to stderr. Its just that the message in the original assertion was wrong since the finalResults.toString gets called before the lambda ever runs. This way we get a similar message but with correct information.

Reply:
generally if you wan't to "log" something then use a logger rather than sys out or sys err